selfapi.js

```diff
@@ -128,11 +133,13 @@ API.prototype = {
     // Export own request handler examples as test functions.
     for (var method in self.handlers) {
       var handler = self.handlers[method];
-      handler.examples.forEach(function (example) {
-        var test =
-          self.testHandlerExample.bind(self, baseUrl, method, handler, example);
-        tests.push(test);
-      });
+      if (handler.examples) {
```

Comment:
If it's all the same to you, I'd prefer making the `examples` key mandatory. I don't think it makes much sense to use Self API without defining any examples. At least there should be an empty `examples: []` entry for later, I think. However, if you still feel that crashing is less useful than simply skipping handlers without examples, please ignore my comment 😄